src/print.js

```diff
@@ -0,0 +1,3 @@
+export default function printMe() {
+
+}
```

Comment:
- [ ] Kindly remove unused files. They don't add functionality to your project and can make your code look messy.